bindings/python/include/aligator/python/visitors.hpp

```diff
@@ -17,10 +17,12 @@ struct ClonePythonVisitor : bp::def_visitor<ClonePythonVisitor<T>> {
   }
 };
 
-template <typename T>
-struct CreateDataPythonVisitor : bp::def_visitor<CreateDataPythonVisitor<T>> {
+template <typename T, typename DataType>
```

Comment:
Maybe we can supply a default template parameter argument: `template<typename T, typename DataType = typename T::Data>`.